### PR TITLE
Create 1.25 Version Release

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

**Comment by pchico83** [comment]
@codyjlandstrom I think you are missing this step:
https://github.com/okteto/docs?tab=readme-ov-file#update-variables

**Comment by codyjlandstrom** [comment]
> @codyjlandstrom I think you are missing this step: https://github.com/okteto/docs?tab=readme-ov-file#update-variables

Thanks for the catch. Fixed

**Comment by pchico83** [comment]
> > @codyjlandstrom I think you are missing this step: https://github.com/okteto/docs?tab=readme-ov-file#update-variables
> 
> Thanks for the catch. Fixed

You need to update the variables file in the 1.25 `versioned` docs too

File: docusaurus.config.js [hunk 1]
```diff
@@ -239,21 +239,26 @@ module.exports = {
...
           versions: {
             current: {
               // aka unreleased version in development
               // Remember to also update "unreleased" redirect if changing the value!
-              label: '1.25',
-              path: '1.25',
+              label: '1.26',
+              path: '1.26',
             },
             1.22: {
```
Comment (ifbyol) [inline docusaurus.config.js:250]: Is this correct? Shouldn't it be `1.25`?

Comment (codyjlandstrom) [reply docusaurus.config.js:250]: This is always one version ahead

Comment (ifbyol) [reply docusaurus.config.js:250]: I refer to the key. It says `1.22` which makes me think it is not right. So, in the `versions`  object, it seems there are elements with the key `1.22`

Comment (codyjlandstrom) [reply docusaurus.config.js:250]: I see what you're saying. It's odd that this compiles just fine. When I change the key to 1.25 it has a failure.
Figuring out what's going on...

**Pull request merged**